openshift/configMap-template.yaml

```diff
@@ -19,22 +19,36 @@ objects:
 - kind: ConfigMap
   apiVersion: v1
   metadata:
-    name: kebechet
+    name: kebechet-configmap
     labels:
       app: kebechet
   data:
-    github-oauth-token: ${GITHUB_OAUTH_TOKEN}
-    kebechet-config: ${KEBECHET_CONFIG}
+    ssh-private-key: ${SSH_PRIVATE_KEY}
+    github-oauth-token: ${KEBECHET_TOKEN}
+    kebechet-config: ${KEBECHET_CONFIGURATION}
+    kebechet-verbose: ${KEBECHET_VERBOSE}
 
 parameters:
+- description: SSH private key to authenticate the public key set in GitHub.
+  displayName: SSH Private Key
+  required: true
+  name: SSH_PRIVATE_KEY
+  value: "ssh_key"
+
 - description: GitHub OAuth token used for authentication.
   displayName: GitHub Oauth token
   required: true
-  name: GITHUB_OAUTH_TOKEN
+  name: KEBECHET_TOKEN
+  value: "token"
```

Comment:
Keep this empty string by default - kebechet will push commits to git repo, but it will not create PRs (and will report that it couldn't create PRs as there is missing token).